ocs_ci/utility/workloads/helpers.py

```diff
@@ -31,3 +47,88 @@ def find_distro(io_pod):
             log.debug(f"Distro is not {distro}")
         else:
             return distro
+
+
+class PgsqlE2E(object):
+    def __init__(self, transactions, pod_name, namespace):
+        self.transactions = transactions
```

Comment:
I think we should discuss again, there is a quite bit of code that can be reused  and this class doesn't have to be in helpers.py,